java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -705,6 +707,338 @@ public static CompletableFuture<Long> arrlen(
         return executeCommand(client, new GlideString[] {gs(JSON_ARRLEN), key});
     }
 
+    /**
+     * Increments or decrements the JSON value(s) at the specified <code>path</code> by <code>number</code>
+     * within the JSON document stored at <code>key</code>.
+     *
+     * @param client The client to execute the command.
+     * @param key The key of the JSON document.
+     * @param path The path within the JSON document.
+     * @param number The number to increment or decrement by.
+     * @return <ul>
+     *     <li>For JSONPath (<code>path</code> starts with <code>$</code>):<br>
+     *         Returns a bytes string representation of an array of bulk strings, indicating the new values after incrementing for each matched <code>path</code>.<br>
```

Comment:
s/bytes string/string/g

Reply:
and don't mention `bulk strings` - it is not java nor GLIDE term (python docs need an update)